stratum/hal/lib/yang/yang_parse_tree.cc

```diff
@@ -176,6 +177,8 @@ void YangParseTree::ProcessPushedConfig(
   AddSubtreeChassis(change.new_config_.chassis());
   // Add all system-related gNMI paths.
   AddSubtreeSystem();
+  // Add all IPsec-related gNMI paths.
+  AddSubtreeIPsec();
```

Comment:
We need to find a way to abstract this out, so as not to burden platforms that don't support IPsec with IPsec extensions.